tests/prepare_test_data.py

```diff
@@ -0,0 +1,179 @@
+import json
+import logging
+import os
+import shutil
+
+import git
+
+from system.plugins.gosymbolextractor.extractor import GoSymbolExtractor
+
+
+# directory where prepared test data will be stored
+# it is expected to be empty, if not, the preparation could fail
+DESTDIR = 'testdata'
+
+# list of projects and their commits to be included in test data
+PROJECTS = [
```

Comment:
It would be better to move it to self-standing json file so it can be easily extended for other projects without modifying the code itself.